cfgmgr/intfmgr.cpp

```diff
@@ -28,6 +30,31 @@ using namespace swss;
 #define LOOPBACK_DEFAULT_MTU_STR "65536"
 #define DEFAULT_MTU_STR 9100
 
+const string WHITESPACE = " \n\r\t\f\v";
+
+inline string rtrim(const string &s)
+{
+    size_t end = s.find_last_not_of(WHITESPACE);
+    return (end == std::string::npos) ? "" : s.substr(0, end + 1);
+}
+
+inline bool enableIpv6Flag(const string &alias)
+{
+    stringstream cmd;
+    string temp_res;
+    cmd << "sysctl -w " << "net.ipv6.conf." << shellquote(alias) << ".disable_ipv6=0";
+    return (swss::exec(cmd.str(), temp_res) == 0) ? true : false;
```

Comment:
Will it be more optimal to write to a file using ifstream instead of spawning a bash shell

Reply:
[](https://github.com/Azure/sonic-swss/blob/master/cfgmgr/intfmgr.cpp#L163) Using bash shell is seemingly the norm in the code. 

Moreover the chances of this race happening are low already, so i think it's okay to use the standard approach

Reply:
Ok